### PR TITLE
Fixes on test files

### PR DESCRIPTION
**Pull request by dulithsenanayake**
This PR includes removal of additional test files, modify cy.intercept and removal of uncaught exceptions from test files.

### CODE REVIEW

**Review by prasa7 — APPROVED** [review]

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -137,10 +137,11 @@ Cypress.Commands.add('deleteUser', (name) => {
 Cypress.Commands.add('deleteApi', (name, version) => {
     cy.visit(`${Utils.getAppOrigin()}/publisher/apis`);
     cy.intercept('**/apis*').as('getApis');
-    cy.wait('@getApis');
-    cy.get('#itest-id-deleteapi-icon-button', { timeout: 30000 });
-    cy.get('#itest-id-deleteapi-icon-button').click();
-    cy.get('#itest-id-deleteconf').click();
+    cy.wait('@getApis', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/support/commands.js:140]: formatting issue

File: tests/cypress/integration/admin/00-add-edit-delete-microgateway-environments.spec.js [hunk 2]
```diff
@@ -34,17 +34,19 @@ describe("Add Edit Delete Microgateway Environments", () => {
         // Wait until the label is saved
         cy.intercept('GET', '**/environments').as('environmentsGet');
         cy.get('button > span').contains('Save').click();
-        cy.wait('@environmentsGet');
-        cy.get('table tr td').contains('MARKETING_STORE').should('exist');
+        cy.wait('@environmentsGet',{timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/00-add-edit-delete-microgateway-environments.spec.js:37]: formatting issue

File: tests/cypress/integration/admin/00-add-edit-delete-microgateway-environments.spec.js [hunk 3]
```diff
@@ -34,17 +34,19 @@ describe("Add Edit Delete Microgateway Environments", () => {
...
 
         // editing
         cy.get('[data-testid="MuiDataTableBodyCell-5-1"] div > div > button:first-child').click();
         cy.get('textarea[name="description"]').type('marketing store');
         // Wait until the label is saved
         cy.intercept('GET', '**/environments').as('environmentsGet');
         cy.get('button > span').contains('Save').click();
-        cy.wait('@environmentsGet');
-        cy.get('table tr td').contains('marketing store').should('exist');
+        cy.wait('@environmentsGet', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/00-add-edit-delete-microgateway-environments.spec.js:47]: formatting issue

File: tests/cypress/integration/admin/01-add-edit-delete-advance-throttle-policies.spec.js [hunk 4]
```diff
@@ -37,10 +37,11 @@ describe("Add Edit Delete advance throttle policies", () => {
         // editing
         cy.intercept('**/throttling/policies/advanced/*').as('getPolicy');
         cy.get('table tr td a').contains(policyName).click();
-        cy.wait('@getPolicy');
-        cy.get('input[name="requestCount"]').clear().type('31');
-        cy.get('button.MuiButton-containedPrimary > span').contains('Update').click();
-        cy.get('table tr td').contains('31').should('exist');
+        cy.wait('@getPolicy', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/01-add-edit-delete-advance-throttle-policies.spec.js:40]: formatting issue

File: tests/cypress/integration/admin/02-add-edit-delete-application-throttle-policies.spec.js [hunk 5]
```diff
@@ -37,11 +37,12 @@ describe("Add Edit Delete application throttle policies", () => {
         // editing
         cy.intercept('**/throttling/policies/application/*').as('getPolicy');
         cy.get(`[data-testid="${policyName}-actions"] > span:first-child`).click();
-        cy.wait('@getPolicy');
-        cy.get('input[name="requestCount"]').clear().type('81');
+        cy.wait('@getPolicy', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/02-add-edit-delete-application-throttle-policies.spec.js:40]: formatting issue

File: tests/cypress/integration/admin/02-add-edit-delete-application-throttle-policies.spec.js [hunk 6]
```diff
@@ -37,11 +37,12 @@ describe("Add Edit Delete application throttle policies", () => {
...
+            cy.get('input[name="requestCount"]').clear().type('81');
+        });
         cy.intercept('GET', '**/throttling/policies/application').as('getPolicies');
         cy.get('button.MuiButton-containedPrimary > span').contains('Save').click();
-        cy.wait('@getPolicies');
+        cy.wait('@getPolicies', {timeout: 3000});
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/02-add-edit-delete-application-throttle-policies.spec.js:45]: formatting issue

File: tests/cypress/integration/admin/03-add-edit-delete-subscription-throttle-policies.spec.js [hunk 7]
```diff
@@ -38,11 +38,12 @@ describe("Add Edit Delete subscription throttle policies", () => {
         // editing
         cy.intercept('**/throttling/policies/subscription/*').as('getPolicy');
         cy.get(`[data-testid="${policyName}-actions"] > a`).click();
-        cy.wait('@getPolicy');
-        cy.get('input[name="requestCount"]').clear().type('10001');
+        cy.wait('@getPolicy', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/03-add-edit-delete-subscription-throttle-policies.spec.js:41]: formatting issue

File: tests/cypress/integration/admin/03-add-edit-delete-subscription-throttle-policies.spec.js [hunk 8]
```diff
@@ -38,11 +38,12 @@ describe("Add Edit Delete subscription throttle policies", () => {
...
+            cy.get('input[name="requestCount"]').clear().type('10001');
+        });
         cy.intercept('GET', '**/throttling/policies/subscription').as('getPolicies');
         cy.get('button.MuiButton-containedPrimary > span').contains('Save').click();
-        cy.wait('@getPolicies');
+        cy.wait('@getPolicies', {timeout: 3000});
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/03-add-edit-delete-subscription-throttle-policies.spec.js:46]: formatting issue

File: tests/cypress/integration/admin/04-add-edit-delete-api-categories.spec.js [hunk 9]
```diff
@@ -33,16 +33,17 @@ describe("Add Edit Delete api categories", () => {
...
         // editing
-        cy.get(`[data-testid="MuiDataTableBodyCell-4-0"] > div > span:first-child`).click();
+        cy.get(`[data-testid="MuiDataTableBodyCell-4-0"] > div > div > span:first-child`).click();
         cy.get('textarea[name="description"]').clear().type('finance apis');
 
         cy.intercept('GET', '**/api-categories').as('getCategories');
         cy.get('button.MuiButton-containedPrimary > span').contains('Save').click();
-        cy.wait('@getCategories');
-        cy.get('[data-testid="MuiDataTableBodyCell-2-0"]').contains('finance apis').should('exist');
+        cy.wait('@getCategories', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/04-add-edit-delete-api-categories.spec.js:41]: formatting issue

File: tests/cypress/integration/admin/05-add-custom-throttle-policies.spec.js [hunk 10]
```diff
@@ -41,8 +41,9 @@ describe("Add custom throttle policies", () => {
 
         cy.intercept('GET', '**/throttling/policies/custom').as('getCustomPolicies');
         cy.get('button.MuiButton-containedPrimary > span').contains('Edit').click();
-        cy.wait('@getCustomPolicies');
-        cy.get('td').contains(secondDesc).should('exist');
+        cy.wait('@getCustomPolicies', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/05-add-custom-throttle-policies.spec.js:44]: formatting issue

File: tests/cypress/integration/admin/09-add-km.spec.js [hunk 11]
```diff
@@ -41,17 +41,17 @@ describe("Add key manager", () => {
         // importing config'
         cy.intercept('**/key-managers/discover').as('importConfig');
         cy.get('button span.MuiButton-label').contains('Import').click();
-        cy.wait('@importConfig');
+        cy.wait('@importConfig', {timeout: 3000}).then(() => {
```
Comment (HiranyaKavishani) [inline tests/cypress/integration/admin/09-add-km.spec.js:44]: formatting issue

**Pull request merged**